compiler_opt/tools/extract_ir_lib.py

```diff
@@ -105,7 +105,9 @@ def thinlto_index_file(self):
 
   def _get_extraction_cmd_command(self, llvm_objcopy_path: str,
                                   cmd_section_name: str):
-    """Call llvm_objcopy to extract the llvmcmd section in self._cmd_file."""
+    """Gets command to make llvm_objcopy extract the llvmcmd section in
+    self._cmd_file.
```

Comment:
took me a bit to understand this, how about "Get llvm-objcopy and process args that, when invoked, would extract the llvmcmd section into the self.cmd_file() file"

or something like that. I was missing what flows where :)